compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUVectorDistribute.cpp

```diff
@@ -184,38 +191,38 @@ class ContractionVectorLayoutOptions : public VectorLayoutOptions {
     if (llvm::any_of(slice, [](Operation *op) {
           return llvm::isa<vector::ContractionOp>(op);
         })) {
-      return;
+      return success();
     }
 
     // TODO: Support masking.
     if (transfer.getMask()) {
-      return;
+      return success();
```

Comment:
This should return failure so that we don't leave this as undistributed. This check also should probably be moved to after the shared memory address space check.

Reply:
makes sense, done.